galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/spi/IKeyStore.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package dev.galasa.framework.spi;
+
+/**
+ * A class to represent a Java Keystore and allow for easy appending and generation
+ * 
+ * @author jamesdavies
+ *
+ */
+public interface IKeyStore {
+	/**
+	 * <p>
+	 * Allows a Base64 encoded PEM cert to be appended to the Keystore
+	 * </p>
+	 * 
+	 * @param cert
+	 * @throws CertificateStoreException
+	 */
+	public void appendPem(String cert) throws CertificateStoreException;
```

Comment:
this would be a straight PEM certificate,   so doesn't need to be base64.   How the manager acquires the PEM certificate may be different to that of the certificate store.

Reply:
Going to need an aliasid for the jks